profiler/options.go

```diff
@@ -52,6 +52,10 @@ type config struct {
 	blockRate     int
 }
 
+func profilingURLForSite(site string) string {
```

Comment:
```suggestion
func urlForSite(site string) string {
```
Will it be enough? I think the "profiling" context is clear from the package...

Also, should we do some validation here to make sure we catch any potential input errors from the user?

Reply:
👍 Sure.

Regarding validation, how strict do you think we want to be? Should we just have a whitelist of sites we compare against? Should we provide constants?

Reply:
I think it should suffice to ensure that the final URL compiles using [`url.Parse`](https://golang.org/pkg/net/url/#Parse) ?

Reply:
👍 That works for me. Good idea.